Source/WebCore/platform/graphics/texmap/coordinated/CoordinatedGraphicsLayerSkia.cpp

```diff
@@ -135,31 +139,101 @@ class AcceleratedBufferPool {
     RunLoop::Timer m_releaseUnusedBuffersTimer;
 };
 
+class UnacceleratedThreadedRenderingPool {
+    WTF_MAKE_FAST_ALLOCATED();
```

Comment:
I don't think we need this form this singleton using NeverDestroyed, because it's not heap allocated. We want WTF_MAKE_NONCOPYABLE, though.

Reply:
Fixed.